### ISSUE TITLE
Request to add the extension of pc8801:Sound Board X(JMB-X1)

### ISSUE THREAD

**Issue by denjhang** [mamedev/mame#8709]
http://upd780c1.g1.xrea.com/pc-8001/j80-25.html
This sound card is actually ym2151 x2 + ay8910 (ym2149)
A sound driver called NRTDRV is currently using this sound card to produce music.

**Comment by angelosa** [comment]
[Known issue](https://github.com/mamedev/mame/blob/master/src/mame/drivers/pc8801.cpp#L22), PC-8001/PC-8801 lacks a bus concept atm.

**Comment by angelosa** [comment]
Tried to quick fit in this one:

```
	map(0xc8, 0xc9).rw(m_opm1, FUNC(ym2151_device::read), FUNC(ym2151_device::write));
	map(0xca, 0xcb).rw(m_opm2, FUNC(ym2151_device::read), FUNC(ym2151_device::write));
	map(0xcc, 0xcd).w(m_ay, FUNC(ym2149_device::address_data_w));
	map(0xcd, 0xcd).r(m_ay, FUNC(ym2149_device::data_r));

	constexpr XTAL jmb_x1_clock = XTAL(4'000'000);

        // TODO: stereo mixing
	YM2151(config, m_opm1, jmb_x1_clock );
	m_opm1->add_route(ALL_OUTPUTS, "mono", 0.50);
	
	YM2151(config, m_opm2, jmb_x1_clock );
	m_opm2->add_route(ALL_OUTPUTS, "mono", 0.50);
	
	YM2149(config, m_ay, jmb_x1_clock / 2);
	m_ay->add_route(ALL_OUTPUTS, "mono", 0.25);
```

With these NTRDRV boots in PC-8801 with no sound produced (requires INT3 coming from master OPM timer) and looks ugly because uses legacy uPD3301 emulation:

![0057](https://user-images.githubusercontent.com/9251271/155852693-98a3bdb9-a6df-425b-972e-6bf0999f2575.png)

while PC-8001 hangs while checking i8214 irq (completely not hooked up there), and suspect it expects the VRTC (notice "fps: 00"). 

![0013x](https://user-images.githubusercontent.com/9251271/155852705-c9f11659-ab36-4f1a-bbd4-5b9cf1b888f5.png)

**Comment by angelosa** [comment]
With referenced commit JMB-X1 works on the tutorial songs that base NRTDRV provides.
![1413](https://user-images.githubusercontent.com/9251271/162455279-0a7169ae-c0bb-4b64-adab-e4b736119808.png)

Not sure what the test utility should do, and how to enable the amber screen seen on the page (is it PC-8001 only?).
Regardless EXP itself needs to be backported to PC-8001mkII, which is the reason about why I'm leaving this open for the time being.

**Comment by denjhang** [comment]
> 

I tried the latest version of MAME and found that NRTDRV's music was played correctly, but all stereo notes became mono.

**Comment by angelosa** [comment]
Yeah that's known: I have no information about the left/right routing for this.

**Comment by angelosa** [comment]
> Regardless EXP itself needs to be backported to PC-8001mkII, which is the reason about why I'm leaving this open for the time being.

Done in https://github.com/mamedev/mame/commit/2e3073c71380148d5bfce176d28874118922e25a , closing
